sippy-ng/src/component_readiness/RegressedTestsModal.js

```diff
@@ -131,6 +131,33 @@ export default function RegressedTestsModal(props) {
       flex: 10,
       renderCell: (param) => <div className="test-name">{param.value}</div>,
     },
+    {
+      field: 'opened',
```

Comment:
There's a hidden field here that could be dependent on whether we're in the default value or not.  We'd need to write something to check if query is the default, it could be done as a follow-up